backend/app/api/endpoints/devices.py

```
@@ -0,0 +1,122 @@
+    """
+    devices = await device_service.get_online_devices(db, current_user.id)
+    return DeviceListResponse(
+        items=[DeviceInfo(**d) for d in devices],
+        total=len(devices),
+    )
+
+
+@router.put("/{device_id}/default")
+async def set_default_device(
+    device_id: str,
+    db: Session = Depends(get_db),
```

Ruff (0.14.14)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)